dlib/test/fft.cpp

```diff
@@ -21,19 +21,53 @@ namespace
     using namespace std;
 
     logger dlog("test.fft");
-
+    static dlib::rand rnd;
+    
+// ----------------------------------------------------------------------------------------
+    
+    static inline int fft_next_fast_size(int n)
```

Comment:
Things inside an anonymous namespace are implicitly static already, so don't declare stuff static as it's unneeded.